src/panoramic/cli/transform/commands.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+from typing import List, Tuple
+
+import click
+from tqdm import tqdm
+
+from panoramic.cli.connections import Connections
+from panoramic.cli.context import get_company_slug
+from panoramic.cli.local.get import get_transforms
+from panoramic.cli.local.writer import FileWriter
+from panoramic.cli.paths import FileExtension, Paths
+from panoramic.cli.print import echo_error, echo_info
+from panoramic.cli.transform.compiler import TransformCompiler
+from panoramic.cli.transform.executor import TransformExecutor
+from panoramic.cli.transform.pano_transform import CompiledTransform, PanoTransform
+
+
+def create_command():
+    echo_info('Scaffolding a new transform...')
+    name = click.prompt('name')
+
+    connections = Connections.load()
+    connection_names = connections.keys() if connections else []
+    connection_base_text = 'connection'
+
+    if len(connection_names) == 0:
+        connection_prompt_text = connection_base_text
+    elif len(connection_names) > 3:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(list(connection_names)[:3])}}},...)'
+    else:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(connection_names)}}})'
+
+    # Assemble target based on input
+    connection = click.prompt(connection_prompt_text)
+
+    target_view_path = click.prompt(f'target: {connection}.', prompt_suffix="")
+    target = f'{connection}.{target_view_path}'
+
+    transform = PanoTransform(name=name, fields=[], target=target)
+    writer = FileWriter()
+    transform_path = Paths.transforms_dir() / f'{transform.name}{FileExtension.TRANSFORM_YAML.value}'
+
+    if Path.exists(transform_path):
+        echo_error(f'Transform {transform_path} already exists')
+    else:
+        writer.write_transform(transform)
+
+
+def exec_command(
+    compile_only: bool = False,
+    yes: bool = False,
+):
+    company_slug = get_company_slug()
+    compiled_transforms: List[Tuple[CompiledTransform, Path]] = []
+
+    transforms_with_path = get_transforms()
+
+    echo_info('Compiling transforms...')
+    with tqdm(transforms_with_path) as compiling_bar:
+        for transform, transform_path in compiling_bar:
+            try:
+                compiled_transform = TransformCompiler.compile(transform=transform, company_slug=company_slug)
```

Comment:
Can we make this an instance method and reuse one compiler for all transforms?

Reply:
Good point. I'll change it to an instance method, which will also address [the comment above ](https://github.com/panoramichq/panoramic-cli/pull/37#discussion_r543198680)

Reply:
Addressed in 9da198e